src/main/fc/fc_tasks.c

```diff
@@ -291,6 +291,12 @@ void fcTasksInit(void)
         } else if (rxConfig()->serialrx_provider == SERIALRX_CRSF) {
             // Reschedule telemetry to 500hz, 2ms for CRSF
             rescheduleTask(TASK_TELEMETRY, TASK_PERIOD_HZ(500));
+        } else if (rxConfig()->serialrx_provider == SERIALRX_SUMD) {
+            // Reschedule telemetry to 1000hz, 1ms for Graupner
+            rescheduleTask(TASK_TELEMETRY, TASK_PERIOD_HZ(1000));
```

Comment:
I suspect that this will negatively affect the maximum possible loop rate. Does this really need to be this fast? I think most users will prefer better flight performance due to faster loop rate to having super fast telemetry.

Also, SUMD and SUMH are RX protocols, and are used by a lot of other hardware that does not use HoTT telemetry. Selecting them should not affect the telemetry rate, this should be done based on the telemetry protocol selection.